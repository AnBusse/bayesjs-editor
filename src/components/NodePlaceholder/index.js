import React from 'react';

const NodePlaceholder = props => (
  <rect
    {...props}
    fill="none"
    stroke="#333"
    strokeWidth="2"
    strokeDasharray="5,5"
  />
);

export default NodePlaceholder;
