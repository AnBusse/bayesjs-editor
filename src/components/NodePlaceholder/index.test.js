import React from 'react';
import { shallow } from 'enzyme';
import NodePlaceholder from './index';

const defaultProps = {
  x: 0,
  y: 5,
  width: 150,
  height: 100,
};

const shallowComponent = (props = {}) => {
  const compProps = { ...defaultProps, ...props };

  return shallow(<NodePlaceholder {...compProps} />);
};

describe('NodePlaceholder Component', () => {
  it('matches snapshot', () => {
    const component = shallowComponent();

    expect(component).toMatchSnapshot();
  });
});
