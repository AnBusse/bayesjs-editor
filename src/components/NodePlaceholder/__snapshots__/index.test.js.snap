// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NodePlaceholder Component matches snapshot 1`] = `
<rect
  fill="none"
  height={100}
  stroke="#333"
  strokeDasharray="5,5"
  strokeWidth="2"
  width={150}
  x={0}
  y={5}
/>
`;
